src/EditorFeatures/Core/SymbolSearch/SymbolSearchUpdateEngine.Update.cs

```diff
@@ -38,12 +38,6 @@ internal partial class SymbolSearchUpdateEngine
         public const string HostId = "RoslynNuGetSearch";
         private const string MicrosoftAssemblyReferencesName = "MicrosoftAssemblyReferences";
 
-        /// <summary>
-        /// Cancellation support for the task we use to keep the local database up to date.
-        /// When VS shuts down it will dispose us.  We'll cancel the task at that point.
-        /// </summary>
-        private readonly CancellationToken _updateCancellationToken;
```

Comment:
This seems wrong to me.  THe idea behind this type is that it's long lived, and unbound to any particular cancellation exception we might have gotten for some operation (like the ones we create during 'add using').  If we don't have this cancellation token, and we now use an ambient cancellatoin token passed in by a feature, we will make it so that this type is no longer long lived, and no longer sits patiently in its loop, updating the index on its prescribed schedule.